hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -2459,6 +2459,24 @@
     <value>0</value>
   </property>
 
+  <property>
+    <description>Provide local directory which includes the related jar file as well as all the dependencies’
+      jar file. We could specify the single jar file or use ${local_dir_to_jar}/* to load all
+      jars under the dep directory.</description>
+    <name>yarn.nodemanager.aux-services.%s.classpath</name>
+    <value>NONE</value>
+  </property>
+
+  <property>
+    <description>Normally, we do not need to set this configuration. The class would be loaded from customized classpath
```

Comment:
Nit: I think saying something like: "Normally this configuration shouldn't be manually set." fit a bit more in the rest of the doc.